shub/settings.py

```diff
@@ -252,6 +278,11 @@ def get_sregistry_envar_list(key):
         STRING_DEFAULTS[key] = value
 
 
+for key in LIST_DEFAULTS:
+    value = get_sregistry_envar_list(key)
+    if value is not []:
+        LIST_DEFAULTS[key] = list(set(value))
```

Comment:
We could take two directions here:

- Just set it here if it's not an empty list.
- update the defaults to include it (and take the unique set of all of those)

What do you think?

Reply:
im not sure what you mean with this?  the defaults have the two lists that I wanted, and they get overridden here by env variables.

Reply:
TLDR you can either do:

```python
# totally override, don't care if something was there
if value:
    LIST_DEFAULTS[key] = value
```
or

```python
# be flexible to allowing to extend
if value:
    LIST_DEFAULTS[key] += value
```
And for the latter (if you do that) I'd do a loop over all of them at the end to do a list(set( to ensure uniqueness.

Reply:
make sense, i kinda moved to the overwrite method, as that is how all the other  settings work.